internal/pkg/levee/testdata/allowpanicontaintedvalues-config.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+AllowPanicOnTaintedValues: true
+Sources:
```

Comment:
Please use the non-RE (string literal) forms as regexp are not needed here. (Unnecessary regexp also exist in `test-config.yaml` and many others, but when that code was written string literal matchers did not exist; since this is new code I believe it would be best if we used the most appropriate matchers possible.)

Reply:
Done.